nncf/quantization/quantize.py

```diff
@@ -26,6 +27,7 @@
 from nncf.parameters import TargetDevice
 
 
+@api(canonical_alias="nncf.quantize")
```

Comment:
@AlexKoff88 check this out - in this way it can be specified that the canonical import name should be "nncf.quantize", and the docs will be generated with the canonical import name in mind. Note that this also requires that the symbol itself is imported in corresponding `__init__.py` file (in this case, `nncf/__init__.py`), otherwise the doc build will fail and the doc build check will be red.

Reply:
I cannot find quantize() in the latest version of the API build, only quantize_with_accuracy_control().

Reply:
Fixed - make sure to reload the page with Ctrl-Shift-R so that you don't get a cached version.

One of the reasons was that had a function `quantize` exposed in `__init__.py` in the same subpackage that also contained the module called `quantize.py`. This meant that the `quantize.py` file was effectively impossible to import from, apart from the exposed `quantize` and `quantize_with_accuracy_control` functions, and this broke down the doc search mechanism, and is anyway considered to be bad practice (since the imported name shadows the subpackage name). Had to rename the `quantize.py` to `quantize_fns.py`, if you have suggestions for a better name, please post them.

Reply:
If only rename `quantize.py` -> `quantize_model.py`